merchants/Personas/PersonasBTCPay.md

```diff
@@ -0,0 +1,44 @@
+
+The most logical way to distinguish merchants in BTCPay Server is to divide them in:
+- Retail Merchants (brick and mortar stores)
+- Online Merchants
+
+### Retail Merchants
+
+### Online Merchants
+
+Online merchants are persona that are using BTCPay Server to accept payments online. Their goal is to either sell a product (physical or digital), a service or accept donations.
+
+#### Bitcoin Enthusiast
+Bitcoin enthusiasts usually deploys his online store for fun and experiment. These users have technical skills and want to try BTCPay Server out. These are small shops or attempts to collect Bitcoin within a certain type of group (Bitcoin Twitter).
+
+#### Medium-size merchants
+These merchants use CMS in most of the cases for their online stores. They use BTCPay via available plugins and easily connect it to their stores.
+
+#### Privacy-orientated merchants
+
+Using BTCPay Server because of the privacy features. These merchants are usually VPN's, hosting providers, VOIP services. [Example here](https://directory.btcpayserver.org/filter/merchants/domains-hosting-vpns/)
+
+#### Enterprise
+
+An enterprise is a business that has large volume of customers. The reason they've implemented BTCPay is most likely problems they've experienced with BitPay BIP70 compatibility or simply a large portion of their customer base requested it. These enterprises commonly use a few Bitcoin payment processors either because they allow more coins or because they've used it in the past. They have developer and support departments that handle customer queries. Their integrations are custom and deeply integrated within their workflow.
+
+Example: [Cheap Air](https://www.cheapair.com/) and their [story](https://www.reddit.com/r/Bitcoin/comments/8ymzpj/update_from_cheapair_now_using_btcpay/), NameCheap their [story](https://www.namecheap.com/about/press-releases/press-releases-info/20-09-21/namecheap_now_offers_bitcoin_payment_option_via_btcpay/) [user flow](https://www.namecheap.com/support/knowledgebase/article.aspx/10312/2200/how-do-i-add-funds-to-my-namecheap-account-using-btcpay). 
+
+#### Charity
+
+Usually an organization that's trying to raise funding for a particular cause, or receive donations in Bitcoin. Larger organizations usually convert their donations to fiat by exchanging their bitcoin on an exchange. These charities have deep understanding of privacy and care about privacy. They don't pay attention to invoices, underpaid amounts or manage their invoices within BTCPay Server. They are okay with whatever amount they receive.
+Example of Charity persona: [The Tor Project](https://donate.torproject.org/cryptocurrency/), [Human Rights Foundation](https://hrf.org/donate-bitcoin/), [Hong Kong Free Press](https://support.hongkongfp.com/)
+
+#### Censored merchants
```

Comment:
I don't know if such merchants exist, but I can imagine in some places accepting money online is difficult - or not possible because of the lack of access to merchant accounts, not due to intentional censorship.

Which of these archetypes would such merchants will fall into?

Reply:
That's a great idea. I would probably form a new group for those type of users, because they don't care probably about the privacy and censorship resistance, they simply do not have any other options. I guess it can in a way be sub-group in Censored merchants, or do you have any other ideas?